samples/MvcSandbox/project.json

```diff
@@ -17,6 +17,9 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
     "Microsoft.Extensions.Logging.Console": "1.0.0-*"
   },
+  "content": [
+    "Views"
+  ],
```

Comment:
I was using the Sandbox to test the bug and my fix in the real world and figured reverting the stuff I did to get it to work in the new world was a waste.